tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -318,7 +332,10 @@ void fillShaderBufferFromLongVectorData(std::vector<BYTE> &ShaderBuffer,
   // enough space.
   const size_t NumElements = TestData.size();
   const size_t DataSize = sizeof(T) * NumElements;
-  ShaderBuffer.resize(DataSize);
+
+  // Ensure the shader buffer is large enough. It should be pre-sized based on
+  // the D3D12_RESOURCE_DESC for the associated D3D12_RESOURCE.
```

Comment:
I should update this comment to be a little more specific about what part of the desc are important